lib/matplotlib/tests/test_text.py

```diff
@@ -30,7 +30,7 @@ def find_matplotlib_font(**kw):
     from matplotlib.font_manager import FontProperties, findfont
     warnings.filterwarnings(
         'ignore',
-        ('findfont: Font family \[u?\'Foo\'\] not found. Falling back to .'),
+        r'findfont: Font family \[u?\'Foo\'\] not found. Falling back to .',
```

Comment:
Err, actually, I don't know what I was thinking here, but the internal quotes still need to be escaped unless using double quotes as @anntzer suggested.